### PR TITLE
add double-conversion CI

### PR DESCRIPTION
**Pull request by borune-k12**
Specify library name and version:  **lib/1.0**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: .github/workflows/double-conversion.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      - 'recipes/double-conversion/**'
+      - 'build.py'
+      - '.github/workflows/double-conversion.yml'
+
+env:
+  CONAN_PASSWORD: ${{ secrets.BintrayApiKey }}
+  IS_PURE_C: true
+
+jobs:
+  openssl:
```
Comment (worldemar) [inline .github/workflows/double-conversion.yml]:
```suggestion
  double-conversion:
```

File: .github/workflows/sqlite3.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
+name: openssl CI
```
Comment (Artalus) [inline .github/workflows/sqlite3.yml]: sqlite3 здесь и ниже
Но вообще, почему здесь, а не отдельным файлом?

File: .github/workflows/sqlite3.yml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+on:
+  pull_request:
+    paths:
+      - 'recipes/openssl/**'
```
Comment (worldemar) [inline .github/workflows/sqlite3.yml]:
```suggestion
      - 'recipes/openssl/all/**'
```

File: .github/workflows/sqlite3.yml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+      - 'build.py'
+      - '.github/workflows/openssl.yml'
+  push:
+    branches:
+      - master
+    paths:
+      - 'recipes/openssl/**'
```
Comment (worldemar) [inline .github/workflows/sqlite3.yml]:
```suggestion
      - 'recipes/openssl/all/**'
```

File: .github/workflows/sqlite3.yml (hunk 2 above)
Comment (worldemar) [reply .github/workflows/sqlite3.yml]: блин а я думал ревью из одного файла... не аппрув!

Comment (borune-k12) [reply .github/workflows/sqlite3.yml]: да, по ошибке залился рецепт для sqlite

File: .github/workflows/double-conversion.yml [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  pull_request:
+    paths:
+      - 'recipes/double-conversion/**'
+      - 'build.py'
+      - '.github/workflows/double-conversion.yml'
+  push:
+    branches:
+      - master
+    paths:
+      - 'recipes/double-conversion/**'
```
Comment (worldemar) [inline .github/workflows/double-conversion.yml]:
```suggestion
      - 'recipes/double-conversion/all/**'
```

File: .github/workflows/double-conversion.yml [hunk 6]
```diff
@@ -0,0 +1,99 @@
+name: double-conversion CI
+
+on:
+  pull_request:
+    paths:
+      - 'recipes/double-conversion/**'
```
Comment (worldemar) [inline .github/workflows/double-conversion.yml]:
```suggestion
      - 'recipes/double-conversion/all/**'
```

**Pull request merged**